src/main/java/io/nats/client/SubscribeOptions.java

```diff
@@ -105,60 +66,35 @@ public long getPullBatchSize() {
         return pull;
     }
 
-
-    private static void checkStreamName(String stream) {
-        if (stream == null || stream.length() == 0 || stream.contains(">") ||
-        stream.contains(".") || stream.contains("*")) {
-            throw new IllegalArgumentException("stream cannot be null, empty, tokenized, or wildcarded");
-        }
-    }
     /**
-     * Setting this enables pull based consumers.  This sets the batch size the server
-     * will send.  Default is 0 (disabled).
-     * @param batchSize number of messages to request in poll. 
+     * Get a default instance of SubscribeOptions
+     * @return the instance
      */
-    public void setPull(long batchSize) {
-        if (batchSize <= 0) {
-            throw new IllegalArgumentException("Batch size must be greater than zero");
-        }
-        pull = batchSize;
+    public static SubscribeOptions getInstance() {

```

Comment:
Can this be renamed to `getDefaultOptions()` or `getDefault()`?

Reply:
getDefaultInstance